packages/react/src/primitives/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { classNameModifierByFlag } from '../shared/utils';
+import { ComponentClassNames } from '../shared/constants';
+import {
+  BaseBreadcrumbItemProps,
+  BreadcrumbsItemProps,
+  ForwardRefPrimitive,
+  Primitive,
+} from '../types';
+import { View } from '../View';
+import { BreadcrumbsContext } from './BreadcrumbsContext';
+
+const BreadcrumbItemPrimitive: Primitive<BreadcrumbsItemProps, 'li'> = (
+  { className, children, isCurrent, ...rest },
+  ref
+) => {
+  const { separator } = React.useContext(BreadcrumbsContext);
+  const componentClasses = classNames(
+    ComponentClassNames.BreadcrumbsItem,
+    classNameModifierByFlag(
+      ComponentClassNames.BreadcrumbsItem,
+      'current',
+      isCurrent
+    ),
+    className
+  );
+  const value = React.useMemo(() => {
+    return { isCurrent };
+  }, [isCurrent]);
+
+  return (
+    <View as="li" className={componentClasses} ref={ref} {...rest}>
+      <BreadcrumbsContext.Provider value={value}>
+        {children}
+      </BreadcrumbsContext.Provider>
+      {isCurrent ? null : (
+        <View
+          as="span"
+          aria-hidden="true"
+          className={ComponentClassNames.BreadcrumbsSeparator}
+        >
+          {separator}
+        </View>
```

Comment:
What if the customer wants to include the `separator` as part of the `children` they pass to `BreadcrumbItem` (meaning no need for a standalone `separator`)?

Reply:
They could do that if they wanted if they pass null to the separator prop and include whatever separator they want in BreadcrumbItem children

Reply:
But the `View` here would still be rendered correct?

Reply:
Ah good point, we could conditionally render this IF there is a separator

Reply:
Just a question here since these comments made me think of it, but if someone wanted to make breadcrumbs look like `/ path / to / file /` they would probably have to do this with `separator=null` and manually prepend the `/` on each item. Then on the current item they would have to add the `/` both before and after?

Reply:
The default `/` is added in the parent `Breadcrumbs`: https://github.com/aws-amplify/amplify-ui/pull/4288/files#diff-38d21fc1975039b31cc31890ae807f45a11e592c8f78104b19a5d5aeb11643a6R17

Reply:
Updated it so there is a `Breadcrumbs.Separator` component, which is the default separator, but could also be used in composition to do `/ path / to / file /` :  https://github.com/aws-amplify/amplify-ui/pull/4288/files#diff-dd142c60c767c65e50f6ed3e579ccd3974c6ce921aa7c73aee8465aec2f3a35b